Jenkinsfile

```diff
@@ -204,8 +204,8 @@ def runLinting() {
   }
   mapParallelTasks['default'] = {
                                 cmd(label: "make check-python", script: "make check-python")
-                                cmd(label: "make check-go", script: "make check-go")
                                 cmd(label: "make notice", script: "make notice")
+                                cmd(label: "make check-go", script: "make check-go")
                                 cmd(label: "Check for changes", script: "make check-no-changes")
```

Comment:
Update order to match `make check`. The `make notice` command can download dependencies, which results in `go.sum` being updated. The `make check-go` seems to clean the `go.sum` again. Depending on order of `make notice` and `make check-go` the lint checks are currently not satisfiable without requiring a fixed order